internal/pkg/task/task.go

```diff
@@ -69,3 +69,15 @@ func convertECSTasks(ecsTasks []*ecs.Task) []*Task {
 	}
 	return tasks
 }
+
+// EarliestStartTime finds the earliest start time from a list of tasks.
+// The `StartedAt` field for the tasks shouldn't be nil.
+func EarliestStartTime(tasks []*Task) time.Time {
```

Comment:
can this be private now? maybe we can move it in the logs.go file since it's the only place where it's used